src/components/StyleGuide/Filtering/ApplyingClearing.mdx

```diff
@@ -0,0 +1,112 @@
+import { Meta } from "@storybook/blocks";
+
+import Link from "../../Link/Link";
+
+<Meta title="Style Guide/Filtering/Applying & clearing filters" />
+
+# Applying and clearing filters
+
+## Table of contents
+
+- {<Link href="#methods" target="_self">Methods</Link>}
+- {<Link href="#applying-filters" target="_self">Applying filters</Link>}
+- {<Link href="#clearing-filters" target="_self">Clearing filters</Link>}
+- {<Link href="#populating-results" target="_self">Populating results</Link>}
+
+## Methods
+
+Filters should be applied and cleared using one of two methods: live filtering or batch filtering.
+
+### Live filtering
+
+- Data is updated in real-time as the user selects filtering options.
+- This is the recommended method for applying filters because:
+  - Users tend to expect and are familiar with real-time filtering due to its
+    prevalence in e-commerce and elsewhere.
+  - We have observed in previous research that users tend to miss `Apply`
+    buttons, particularly when they sit out of sight on the page.
+  - Batch filtering comes at the risk of returning zero results after which
+    users have to engage in a trial-and-error process to get useful results.
+
+### Batch filtering
+
+- The user selects multiple filtering options and the data is updated after the
+  user takes the additional action of clicking the "Apply filters" button
+  included in the `filter bar`.
+- This method is recommended for:
+  - very heavy data sets
+  - low performing apps
```

Comment:
I think we should define what "low performing" means. If we can't right now, "very heavy data sets" is enough and we should remove "low performing apps".

Reply:
I'm thinking, if we can get a baseline of web core vital metrics, for example, we can say something along the lines of, "does not meet Digital performance baseline metrics", or something to that effect.